codeclimate-validate-config/Gemfile.lock

```diff
@@ -0,0 +1,49 @@
+GEM
```

Comment:
I'm not sure it's best for us to split commands to the point where they have entirely different dependency trees. That seems like a good way to some day introduce bugs & inconsistent behavior from updating some commands and not others.

FWIW, git doesn't structure itself this way: dependencies are "global".